erizo_controller/erizoJS/adapt_schemes/notify-break-recover.js

```diff
@@ -101,9 +100,9 @@ exports.MonitorSubscriber = function (log) {
                     }
                     break;
                 case BW_RECOVERING:
-                    log.debug("In RECOVERING state lastValue", lastBWValue, "lastAverage", lastAverage, "lowerThres", wrtc.lowerThres);
+                    log.debug("message: trying to recover, id: " + wrtc.wrtcId + ", state:BW_RECOVERING, lastBandwidthValue: " + lastBWValue + ", lastAverageBandwidth: " + lastAverage + ", lowerThreshold: " + wrtc.lowerThres);
```

Comment:
`log.debug("message: trying to recover, id: " + wrtc.wrtcId + ", state: BW_RECOVERING, lastBandwidthValue: " + lastBWValue + ", lastAverageBandwidth: " + lastAverage + ", lowerThreshold: " + wrtc.lowerThres);`